INSS.EIIR.Web/Views/SearchDetails/Index.cshtml

```diff
@@ -105,29 +106,36 @@
                                     }
                                 </dd>
                             </div>
+                            @if (Model.CaseDetails.insolvencyType == INSOLVENCY_TYPE.DRO || Model.CaseDetails.insolvencyType == INSOLVENCY_TYPE.BANKRUPTCY)
+                            {
+                                <div class="govuk-summary-list__row">
+                                    <dt class="govuk-summary-list__key"> Case description </dt>
+                                    <dd class="govuk-summary-list__value"> @Html.DisplayFor(model => Model.CaseDetails.caseDescription) </dd>
```

Comment:
why CaseDetails.caseDescription is camelCase instaed of PascalCase

Reply:
Its an existing model, please refer CaseDetails